features/build_runner/fastlane_build_runner.rb

```diff
@@ -7,6 +7,7 @@ module FastlaneCI
   # fastlane Fastfile configurations
   # TODO: run method *should* return an array of artifacts
   class FastlaneBuildRunner < BuildRunner
+    include FastlaneCI::Logging
```

Comment:
Uhhh, this will cause lots of conflicts, as I'm refactoring `fastlane_build_runner`, `build_runner` and `build_runner_service`